ramls/event_config.raml

```diff
@@ -12,6 +12,7 @@ documentation:
 types:
   errors: !include raml-util/schemas/errors.schema
   eventEntity: !include event_entity.json
+  eventEntries: !include event_entries.json
```

Comment:
rename event_entries to event_entity_collection to make it more clear how entity is connected with entity_collection

Reply:
thanks for the comments!
Done